core/components/TabPanel.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+export const TabPanel = (props) => {
+  const { value, index, children } = props
+  const useStyles = makeStyles(() => ({
+    tab: {
+      '& .MuiBox-root': {
+        padding: '0px'
+      }
+    }
+  }))
+
+  const classes = useStyles()
+
+  return (
+    <div
+      className={classes.tab}
```

Comment:
อันนี้ใช้ className ของ tailwind ไม่ได้หรอ ทำไมต้องใช้ style ของ material อ่ะ? หรือถ้าไม่ได้ก็ลอง inline ได้ม้า?
```suggestion
      className="m-0"
```